src/test/java/picard/vcf/GenotypeConcordanceTest.java

```diff
@@ -572,4 +575,20 @@ public void testNoCallVariants() {
 
         Assert.assertEquals(genotypeConcordance.instanceMain(new String[0]), 0);
     }
+
+    @Test
+    public void testNormalizeAllelesForWritingVCF() {
+        final File truthVcfPath = new File(TEST_DATA_PATH.getAbsolutePath(), NORMALIZE_NO_CALLS_TRUTH);
+        final File callVcfPath  = new File(TEST_DATA_PATH.getAbsolutePath(), NORMALIZE_NO_CALLS_CALL);
+
+        final GenotypeConcordance genotypeConcordance = new GenotypeConcordance();
+        genotypeConcordance.TRUTH_VCF = truthVcfPath;
+        genotypeConcordance.TRUTH_SAMPLE = "truth";
+        genotypeConcordance.CALL_VCF = callVcfPath;
+        genotypeConcordance.CALL_SAMPLE = "truth";
+        genotypeConcordance.OUTPUT = new File(OUTPUT_DATA_PATH, "MultipleRefAlleles");
+        genotypeConcordance.OUTPUT_VCF = true;
+
+        Assert.assertEquals(genotypeConcordance.instanceMain(new String[0]), 0);
```

Comment:
can you also read in the metrics file and validate the results in it, rather than just checking that the program didn't blowup?